packages/web-components/src/components/ic-side-navigation/test/basic/ic-side-navigation.e2e.ts

```diff
@@ -770,5 +772,73 @@ describe("ic-side-navigation", () => {
 
       expect(labelVisibility).toBe("visible");
     });
+
+    it("renders navigation item labels when using slotted app-title and side nav is expanded", async () => {
+      const page = await newE2EPage();
+      await page.setContent(icSideNavigationAppTitle);
+
+      let visibleLabel = await page.evaluate(() => {
+        const label = document
+          .querySelector("ic-side-navigation ic-navigation-item")
+          .querySelector("ic-typography");
+
+        return window.getComputedStyle(label).visibility;
+      });
+
+      expect(visibleLabel).toBe("hidden");
+
+      const menuExpandBtn = await page.find(
+        "ic-side-navigation >>> .side-navigation button.menu-expand-button"
+      );
+
+      await menuExpandBtn.click();
+
+      await page.waitForTimeout(ANIMATION_MS_THRESHOLD);
+
+      visibleLabel = await page.evaluate(() => {
+        const label = document
+          .querySelector("ic-side-navigation ic-navigation-item")
+          .querySelector("ic-typography");
+
+        return window.getComputedStyle(label).visibility;
+      });
+
+      expect(visibleLabel).toBe("visible");
+    });
+
+    it("renders visible labels when slotted menu items and expanded", async () => {
```

Comment:
```suggestion
    it("renders visible labels when slotted menu items and side nav is expanded", async () => {
```
For consistency thought to add this :)